_includes/tutorials/time-concepts/ksql/markup/dev/transient-query-logtime.adoc

```diff
@@ -0,0 +1,19 @@
+Let’s inspect the events in this newly created `TEMPERATURE_READINGS_LOGTIME` stream by running a `SELECT` statement with an `EMIT CHANGES` clause, limited to 10.
+It shows the payload fields `TEMPERATURE` and `EVENTTIME`, plus the `ROWTIME`.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/time-concepts/ksql/code/tutorial-steps/dev/transient-query-logtime.sql %}</code></pre>
++++++
+
+This should yield the following output:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/time-concepts/ksql/code/tutorial-steps/dev/expected-transient-query-logtime.log %}</code></pre>
++++++
+
+Notice that for each row:
+
+- The `EVENTTIME` value in ksqlDB corresponds exactly to the `payload eventTime` set by the producer
+- The `ROWTIME` value in ksqlDB corresponds exactly to the `log timestamp` printed by the producer's callback (i.e., ingestion time, when the record was written into the broker's log file)
+
+All processing on this stream is based on `ROWTIME`, consequently this results in processing based on ingestion time.
```

Comment:
Might be good to mention here that `ROWTIME` is a pseudo column and is the Kafka record timestamp

Reply:
Agree and resolved